server/github.go

```diff
@@ -300,3 +308,19 @@ func (s *Server) waitForStatus(ctx context.Context, pr *model.PullRequest, statu
 		}
 	}
 }
+
+func (s *Server) GetStatus(pr *model.PullRequest, statusContext string) (status *github.RepoStatus, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), requestTimeoutGithub)
+	defer cancel()
+	statuses, _, err := s.GithubClient.Repositories.ListStatuses(ctx, pr.RepoOwner, pr.RepoName, pr.Sha, nil)
```

Comment:
Similar comment as above: Let's just go with a high `PerPage` number instead of proper paging.

Reply:
No need for this method anymore.